src/plugins/vis_type_timeseries/public/application/components/lib/index_pattern_select/index_pattern_select.tsx

```diff
@@ -98,25 +96,6 @@ export const IndexPatternSelect = ({
     });
   }, [fetchedIndex]);
 
-  useEffect(() => {
```

Comment:
this effect was moved into parent component to reuse `fetchedIndex` for field selector placeholder